apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -1179,105 +1179,119 @@ export class PretixPipeline implements BasePipeline {
 
-      //let didUpdate = false;
-
-      return sqlQueryWithPool(this.context.dbPool, async (client) => {
-        // for (const e of emails) {
-        //   didUpdate =
-        //     didUpdate ||
-        //     (await this.consumerDB.save(
-        //       client,
-        //       this.id,
-        //       e.email,
-        //       semaphoreId,
-        //       new Date()
-        //     ));
-        // }
-
-        // const provider = this.autoIssuanceProvider;
-        // if (provider) {
-        //   const newManualTickets = (
-        //     await Promise.all(
-        //       emails.map(async (e) =>
-        //         provider.maybeIssueForUser(
-        //           e.email,
-        //           await this.getAllManualTickets(client),
-        //           await this.db.loadByEmail(this.id, e.email)
-        //         )
-        //       )
-        //     )
-        //   ).flat();
-
-        //   await Promise.allSettled(
-        //     newManualTickets.map((t) =>
-        //       this.manualTicketDB.save(client, this.id, t)
-        //     )
-        //   );
-        // }
+      await sqlQueryWithPool(this.context.dbPool, async (client) => {
+        let didUpdate = false;
+        for (const e of emails) {
+          didUpdate =
+            didUpdate ||
+            (await this.consumerDB.save(
+              client,
+              this.id,
+              e.email,
+              semaphoreId,
+              new Date()
+            ));
+        }
 
         // If the user's Semaphore commitment has changed, `didUpdate` will be
         // true, and we need to update the Semaphore groups
-        // if ((this.definition.options.semaphoreGroups ?? []).length > 0) {
-        //   if (didUpdate) {
-        //     span?.setAttribute("semaphore_groups_updated", true);
-        //     await this.triggerSemaphoreGroupUpdate(client);
-        //   }
-        // }
-
-        const tickets = (
-          await Promise.all(
-            emails.map((e) =>
-              this.getEdDSATicketsForEmail(client, e.email, semaphoreId)
-            )
-          )
-        ).flat();
-
-        span?.setAttribute("pcds_issued", tickets.length);
-
-        const actions: PCDAction[] = [];
-
-        if (await this.db.hasLoaded(this.id)) {
-          actions.push({
-            type: PCDActionType.DeleteFolder,
-            folder: this.definition.options.feedOptions.feedFolder,
-            recursive: true
-          });
+        if ((this.definition.options.semaphoreGroups ?? []).length > 0) {
+          if (didUpdate) {
+            span?.setAttribute("semaphore_groups_updated", true);
+            await this.triggerSemaphoreGroupUpdate(client);
+          }
         }
+      });
 
-        const ticketPCDs = await Promise.all(
-          tickets.map((t) => EdDSATicketPCDPackage.serialize(t))
-        );
-
-        if (this.definition.options.enablePODTickets) {
-          const podTickets = (
+      // Is this still needed?
```

Comment:
why are we commenting out auto-issuance / manual tickets? i'm worried if there's some unintended consequence

Reply:
I wasn't sure if it was still needed, but I've re-enabled it as well.